resource-definitions/terraform-driver/private-git-repo/ssh-secret-refs.tf

```diff
@@ -0,0 +1,35 @@
+resource "humanitec_resource_definition" "example-resource" {
+  driver_type = "humanitec/terraform"
+  id          = "example-resource"
+  name        = "example-resource"
+  type        = "some-resource-type"
+
+  driver_inputs = {
+    # This examples uses secret references, pointing at a secret store
+    # to obtain the actual values
+    secret_refs = jsonencode({
+
+      source = {
+        # Using the ssh_key for a connection to the Git repo via SSH
+        ssh_key = {
+          ref   = var.ssh_key_secret_reference_key
+          store = var.secret_store
+        }
+      }
+
+      variables = {
+        # ...
+      }
+    })
+
+    values_string = jsonencode({
+      # Connection information to the target Git repo
+      source = {
+        path = var.tf_module_github_path
```

Comment:
To be consistent with the `s3` example in this PR, and to be more explicit for our end users, let's change these `var`s by:
```
          path = "s3/terraform/bucket/"
          rev  = "refs/heads/main"
          url  = "https://my-domain.com/my-org/my-repo.git"
```

With appropriate `url` if ssh.

In all the different places in this PR too.

Thanks!

Reply:
Makes sense, thanks! I pushed changes to that end.